move/axelar_gateway/sources/types/weighted_signers.move

```diff
@@ -47,6 +89,38 @@ public(package) fun nonce(self: &WeightedSigners): Bytes32 {
     self.nonce
 }
 
+/// -----
+/// Internal Functions
+/// -----
+
+/// Validates the order of the signers.
+/// The signers must be in ascending order by their public key.
+/// Otherwise, the error `EInvalidOperators` is raised.
+fun validate_signers_order(signers: vector<WeightedSigner>) {
+    let mut previous = weighted_signer::default();
+    signers.do!<WeightedSigner>(
+        |signer| {
+            signer.validate(&previous);
```

Comment:
`validate` method should only take `self` instead of another signer. Use `lt` comparison separately to check. I'm also fine with just checking the weight here explicitly.